### PR TITLE
start work on /help-wanted/ page

### LINKED ISSUE #192: Feed of "help wanted" issues
In order to maximize the visibility of "help wanted" labels in rOpenSci packages and promote cross-support within the community, I think it would be helpful to have a process that consumes the feed of rOpenSci and rOpenSciLabs issues, filtering to only those that are tagged "help wanted" (and "help-wanted"), and

- Generates a page of recent help wanted issues (perhaps a column at https://ropensci.org/packages/).
- Feeds a Slack channel with alerts (#help-wanted-issues)
- Runs a twitterbot (@ropensci_help ?)
- Creates summaries suitable for a newsletter section ("Recent help requests")

### PR DESCRIPTION
**Pull request by maelle**
Fix #192 

TODOS

* [x] Fetch JSON from URL. 
* [x] Add docs of this in the README.
* [x] Link the page from the navbar under packages.
* [ ] Link from other places in particular community page.

Some notes

* The JSON I am using is a fake one. I assume we'd have a script to run to update it. I wonder whether it should be automatic (I am now thinking it'd be fine to not clean the data if the page included some disclaimer). In all cases, we should probably create renv files for the different R tasks in roweb3 (the use cases, the help wanted issues, the newsletter). Ideally the script and renv file would live in roweb3?
* At the top of the page, the cards are three random issues.
* I don't use the updated field at the moment.
* Should there also be a RSS and JSON feeds one could actually subscribe to?

Next steps:

@steffilazerte I'll take any comment on the page display / where it's linked from + on the workflow for updating the JSON. We also need to replace "BLABLABLA".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ Note that main.min.css that contains all the CSS is stored under themes/ropensci
 * Start a new post with the newsletter archetype, use the same title "Latest rOpenSci News Digest" and slug "ropensci-news" and tag "newsletter".
 * **Make sure you have all dependencies from roweb3 DESCRIPTION.**
 * **Make sure your branch is based on the latest commit of the default branch, re-base if needed.**
-* Run `get_use_cases.R` to update use cases data (you need a Discourse API key, contact Scott).
+* Open the Rproj in `scripts/use-cases`, run `renv::restore()` and source `get_use_cases.R` to update use cases data (you need a Discourse API key, contact Scott).
```
Comment (maelle) [inline README.md:212]: I'm still not entirely sure having an Rproj inside an Rproj is the best idea but I think we do need separate renv files for use cases / newsletter / help wanted?

File: data/help/issues.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "title": "Advertisting: answer to open question about opening binary files",
+    "labels": ["good first issue", "help wanted"],
+    "author": "Hugo Gruson",
```
Comment (maelle) [inline data/help/issues.json:5]: this will be empty was some username have no name.

**Comment by maelle** [comment]
Direct link https://deploy-preview-212--ropensci.netlify.app/help-wanted/

File: themes/ropensci/layouts/help-wanted/list.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{{ define "main" }}
+ <section class="section section-generic">
+      <div class="container">
+        <!-- Header text -->
+        <div class="row">
+          <div class="col-md-8">
+            <h1>Help Wanted</h1>
+            <p class="subtitle">
+              If you were looking for issues where your expertise is needed, search no more! BLABLABLA. See also our <a href = "https://contributing.ropensci.org/">contributing guide</a> for finding other ways to contribute to rOpenSci.
```
Comment (steffilazerte) [inline themes/ropensci/layouts/help-wanted/list.html:9]:
```suggestion
              If you were looking for issues where your expertise is needed, search no more! This list features issues for rOpenSci packages specifically labelled as "Help wanted" (package developer is actively soliciting help). If you're just getting started with making contributions, check out the issues labelled "Good first issue", as a way to ease into the process. Remember that not all issues require code contributions, sometimes it's documentation, use cases, vignettes, etc.
              <br>See also our <a href = "https://contributing.ropensci.org/">contributing guide</a> for finding other ways to contribute to rOpenSci.
```

**Review by steffilazerte — COMMENTED** [review]
I think this is great! I've added some comments, and made a suggestion for filling in the blahblahblah 😁 

For the JSON, you can either adapt my comms/content-n-programs/twitter-promo/issues/issues.R script, or perhaps I should create a mini package for grabbing use cases and help-wanted issues that could be used here, and then in in my twitter-promo section. You did suggest making a small package for my utils scripts, so perhaps this would be the way to go?

**Comment by maelle** [comment]
@steffilazerte what's your take on 

> Should there also be a RSS and JSON feeds one could actually subscribe to?

**Comment by steffilazerte** [comment]
> @steffilazerte what's your take on
> 
> > Should there also be a RSS and JSON feeds one could actually subscribe to?

I think it's overkill, especially as we'll be tweeting as well.

**Comment by maelle** [comment]
Ok, thanks @steffilazerte!

I'll wait for the Netlify URL before working on this again.

**Review by steffilazerte — COMMENTED** [review]
So exciting! I'm still at a loss how the Netlify all worked out but at least it did, thanks!

I have a couple of suggestions, but only if you agree, I definitely don't feel that strongly about them :grin: Can't wait to see this go live!

**Pull request merged**